modules.d/35network-legacy/99-default.link

```diff
@@ -0,0 +1,8 @@
+# This file is part of dracut network-legacy module.
```

Comment:
Why don't we take the systemd shipped one, so we leverage all the potential new options?

Reply:
Because you cannot guarantee that systemd is installed on those alternative init systems

Reply:
For the legacy network stuff it probably should be a udev rule not an .link file so this works on alternative init systems ( silly to be placing it under /usr/lib/systemd/network/ since systemd does not exist on those systems )